includes/Admin/Editor.php

```diff
@@ -355,6 +366,8 @@ public function get_editor_settings(): array {
 		// See https://github.com/GoogleForCreators/web-stories-wp/issues/10809.
 		$publisher_name = html_entity_decode( $story->get_publisher_name(), ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML401 );
 
+		$shopping_provider = $this->settings->get_setting( $this->settings::SETTING_NAME_SHOPPING_PROVIDER );
```

Comment:
What if someone filtered `web_stories_shopping_vendors` to use a custom shopping provider?

Reply:
Never mind, forgot that they would then select the custom provider in the settings.